libs/features/pages/booklets-page/src/lib/components/booklets-hero-section/booklets-hero-section.component.scss

```diff
@@ -0,0 +1,71 @@
+.booklets-page{
+    position: relative;
+    background-color: var(--elewa-website-booklets-hero-section-background-color);
+    height: 100vh;
+    overflow: hidden; 
+
+    .booklets-text {
+        display: flex;
+        position: absolute;
+        justify-content: center;
+        width: 60%;
+        margin-left: 50px;
+    }
+
+    .booklets-button{
+        margin-top: 420px;
+        margin-left: 60px;
+    }
+    
+    .curved-shape {
+        position: absolute;
+        bottom: 0;
+        left: 35%; 
+        right: 0;
+        max-width: 70%;
+        height: 400px; 
+        background: transparent linear-gradient(111deg, var(--elewa-website-booklets-hero-section-curve-color) 0%, transparent 100%) 0% 0% no-repeat padding-box;
+        opacity: 0.87;
+        clip-path: polygon(
+            0% 100%, 100% 100%, 100% 20%, 97% 16%, 94% 12%, 92% 10%, 90% 8%, 88% 6%, 86% 4%, 84% 3%, 82% 2%, 80% 1%, 78% 1%, 76% 0%, 74% 1%, 72% 1%, 70% 2%, 68% 3%, 66% 4%, 64% 6%, 62% 8%, 60% 10%, 58% 12%, 56% 16%, 54% 20%, 52% 22%, 50% 26%, 48% 30%, 46% 34%, 44% 38%, 42% 42%, 40% 46%, 38% 50%, 36% 54%, 34% 58%, 32% 62%, 30% 66%, 28% 70%, 26% 74%, 24% 78%, 22% 80%, 20% 84%, 18% 88%, 16% 92%, 14% 94%, 12% 96%, 10% 97%, 8% 98%, 6% 99%, 4% 99%, 2% 100%, 0% 100%
+        );
+        shape-outside: polygon(
+            0% 100%, 100% 100%, 100% 20%, 97% 16%, 94% 12%, 92% 10%, 90% 8%, 88% 6%, 86% 4%, 84% 3%, 82% 2%, 80% 1%, 78% 1%, 76% 0%, 74% 1%, 72% 1%, 70% 2%, 68% 3%, 66% 4%, 64% 6%, 62% 8%, 60% 10%, 58% 12%, 56% 16%, 54% 20%, 52% 22%, 50% 26%, 48% 30%, 46% 34%, 44% 38%, 42% 42%, 40% 46%, 38% 50%, 36% 54%, 34% 58%, 32% 62%, 30% 66%, 28% 70%, 26% 74%, 24% 78%, 22% 80%, 20% 84%, 18% 88%, 16% 92%, 14% 94%, 12% 96%, 10% 97%, 8% 98%, 6% 99%, 4% 99%, 2% 100%, 0% 100%
```

Comment:
This creates a very off background. All you need is to create a smooth pill-shaped rectangle and then transform it then cut out the overflow.